src/v/redpanda/application.cc

```diff
@@ -384,7 +384,7 @@ int application::run(int ac, char** av) {
     std::cout << community_msg << std::endl;
 
     return app.run(ac, av, [this, &app] {
-        vlog(_log.info, "Redpanda {}", redpanda_version());
+        vlog(_log.info, "Redpanda version {}", redpanda_version());
```

Comment:
nit: `Redpanda v23.2` reads nice. So does `Redpanda version: v23.2`. However, `Redpanda version v22.3`.

First reads a product name. Second reads as a definition. The third one doesn't have any structure to me.

:D